Example/TABTestKit/Base.lproj/Main.storyboard

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14490.70" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="VEl-fI-I9x">
```

Comment:
As we still supporting Xcode 10. Might be great to redo in Xcode 10 the change until we drop Xcode 10 at some point.